.github/workflows/tests.yml

```diff
@@ -14,32 +14,23 @@ jobs:
       uses: actions/checkout@master
 
     - name: Set up Homebrew
-      run: |
-        if which brew &>/dev/null; then
-          HOMEBREW_PREFIX="$(brew --prefix)"
-          HOMEBREW_REPOSITORY="$HOMEBREW_PREFIX/Homebrew"
-          brew update-reset "$HOMEBREW_REPOSITORY"
-          ln -s "$PWD" "$HOMEBREW_REPOSITORY/Library/Taps/homebrew/homebrew-formula-analytics"
-        else
-          HOMEBREW_PREFIX=/home/linuxbrew/.linuxbrew
-          HOMEBREW_REPOSITORY="$HOMEBREW_PREFIX/Homebrew"
-          sudo mkdir -p $HOMEBREW_PREFIX
-          sudo git clone --depth=1 https://github.com/Homebrew/brew "$HOMEBREW_REPOSITORY"
+      id: set-up-homebrew
+      uses: Homebrew/actions/setup-homebrew@master
 
-          sudo mkdir -p "$HOMEBREW_REPOSITORY/Library/Taps/homebrew"
-          sudo ln -s "$PWD" "$HOMEBREW_REPOSITORY/Library/Taps/homebrew/homebrew-formula-analytics"
+    - name: Cache Homebrew Bundler RubyGems
+      id: cache
+      uses: actions/cache@master
```

Comment:
```suggestion
      uses: actions/cache@main
```

Reply:
Would rather leave these as-is until they start failing/complaining and we update them all en-masse.

Reply:
They already show a warning.

Reply:
Where?

<img width="777" alt="Screenshot 2020-08-05 at 12 30 09" src="https://user-images.githubusercontent.com/125011/89407849-6cee5600-d717-11ea-98c4-ab59c8c0dcac.png">

Reply:
Hmm, I definitely saw a warning for the `checkout` action in the cask repo, otherwise I wouldn't even have brought it up.

Reply:
https://github.com/Homebrew/homebrew-cask/actions/runs/184608720

Reply:
Yeh, looks like they are migrating one-by-one 😬

Reply:
But we also have `checkout@master` in this workflow and it doesn't show anything … 😕

Reply:
I think all `actions/*` have `main` now and are no longer updating `master`.

We've had the `setup-ruby` warning on Homebrew/brew: https://github.com/Homebrew/brew/pull/8090. It's a bit weird how it's not consistently warning for all actions in all repos.

Reply:
Yeh, very weird.